### PR TITLE
Term entry c sharp sort

### LINKED ISSUE #1498: [Term Entry] C# Array `.Sort()`
### Reason/inspiration (optional)

We would like a new term entry for [C#](https://www.codecademy.com/resources/docs/c-sharp) about the Array.Sort() method.

The entry should go in a new file in docs/content/c-sharp/concepts/arrays/terms/sort/sort.md

The entry should

Have a syntax section illustrating the method with a \`\`\`pseudo code block.
An example showing the method in use with a \`\`\`cs code block.
A runnable codebyte example in a \`\`\`codebyte/csharp block.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- Added new entry for C# Array methods: .Sort()
- Closes #1498 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### Additional information

I am using npm; I have node 18.12.
I was unable to run format:verify (prettier is not recognized)
I was unable to run lint:md (markdownlint is not recognized)
I was unable to run test (jest is not recognized)

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@cslylla, Thanks for the good entry! I've done a first review and don't have any comments. I'm passing it straight on to a second review.

File: content/c-sharp/concepts/arrays/terms/sort/sort.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+```pseudo
+Array.Sort(myArray);
+```
+
+`.Sort()` is a static method of the `Array` object. It takes one parameter, `myArray`, a one-dimensional zero-based array. The `.Sort()` method arranges the elements of the given array in ascending or alphabetical order.
+
+## Example
+
+The following example initializes the `letters` array with strings, then uses the `.Sort()` method to arrange the strings alphabetically. Finally, the `Console.Write()` method prints the sorted array to the console with a `foreach` loop:
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/arrays/terms/sort/sort.md:26]:
```suggestion
The following example initializes the `letters` array with strings, then uses the `.Sort()` method to arrange the strings alphabetically. Finally, a `foreach` loop passes each element of the sorted array to the `Console.Write()` method to be printed:
```

Comment (caupolicandiaz) [reply content/c-sharp/concepts/arrays/terms/sort/sort.md:26]: Just a minor edit here, the original text suggests that the console.write method is ahead of the loop in the order of operations

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cslylla overall very clean, just one small comment

File: content/c-sharp/concepts/arrays/terms/sort/sort.md (hunk 1 above)
Comment (cslylla) [reply content/c-sharp/concepts/arrays/terms/sort/sort.md:26]: Yes, that is much more precise!

**Review by caupolicandiaz — APPROVED** [review]
@cslylla Looks good

**Comment by yangc95** [comment]
@cslylla live link: https://www.codecademy.com/resources/docs/c-sharp/arrays/sort

All set!
Screen Shot 2023-02-04 at 7 39 50 PM

cc: @caupolicandiaz @SSwiniarski @KTom101

**Pull request merged**